README

```diff
@@ -12,15 +12,15 @@
 This program is a simple translations editor for PO and XLIFF files. It also
 serves as a GUI frontend to more GNU gettext utilities (win32 version
 is part of the distribution) and catalogs editor/source code parser. It helps
-with translating application into another language. For details on principles
+with translating applications into another language. For details on principles
 of the solution used, see GNU gettext documentation or wxWidgets' wxLocale
 class reference.
 
 
  Installing
 ============
 
-Easily-installable prebuilt binaries are provided for Windows and macOS and
+Easily-installable prebuilt binaries are provided for Windows and macOS are
```

Comment:
The second "are" in the sentence is not correct yet.
Easily-installable prebuilt binaries are provided for Windows and macOS **are** available from https://poedit.net/download.

How about a shortened version:
Easily-installable prebuilt binaries for Windows and macOS are available from https://poedit.net/download.

Reply:
Right, that makes sense